scenario14/jmeter/01-Scenario14-EnforcePasswordReset.jmx

```diff
@@ -65,19 +74,19 @@
             <stringProp name="Argument.name">roleName</stringProp>
             <stringProp name="Argument.value">${__property(roleName)}</stringProp>
             <stringProp name="Argument.metadata">=</stringProp>
-            <stringProp name="Argument.desc">sol14role</stringProp>
+            <stringProp name="Argument.desc">sol14UserRole</stringProp>
```

Comment:
Please parameterize all the values without hardcoding

Reply:
Thanks for the feedback, the script is parameterized, what we see as "sol14UserRole" is only a sample value which comes under description column